.gitattributes

```diff
@@ -13,3 +13,4 @@
 /tests export-ignore
 /.travis.yml export-ignore
 /UPGRADE.md export-ignore
+/ci export-ignore
```

Comment:
Please use alphabetical order (before `/composer.lock`).

Reply:
Sure. I've updated it.